chjcoder/boj20529/boj20529.py

```diff
@@ -0,0 +1,30 @@
+from itertools import combinations
+def mbti_dis(lst):
+    dis = 8
+    dic = {}
+    for mbti in lst:   # 딕셔너리에 저장
+        if mbti not in dic.keys():
+            dic[mbti] = 1
+        else:
+            dic[mbti] += 1
+            if dic[mbti] == 3: # 같은 MBTI가 3개가 되면 거리 0
+                dis = 0
+                return dis
+    # 같은 MBTI 2개 이하
+    cnt = 0
+    for mbti in combinations(lst,3): #[(1,2,3),(3,4,5)]
+        cnt = 0
+        for mbti_xy in combinations(mbti,2):
+            x,y = mbti_xy
+            for i in range(4):
+                if x[i] != y[i]:
+                    cnt += 1
```

Comment:
unpack과 zip을 활용하면 불필요한 할당 라인이 줄어들 것 같아요
```suggestion
        for mbti_xy in combinations(mbti,2):
            for x, y in zip(*mbti_xy):
                cnt += xi != yi
```

Reply:
arg unpacking은 잘 활용하지 않았는데 고려해보겠습니다! 감사합니다!!